base-image/startscript.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/sh
```

Comment:
And to be safer, we should probably add a `set -eu` to avoid unexpected behaviours for missing variables or continued execution over a previous failure.